cmd/tempo-cli/cmd-list-block.go

```diff
@@ -70,17 +70,20 @@ func dumpBlock(r tempodb_backend.Reader, c tempodb_backend.Compactor, tenantID s
 
 	unifiedMeta := getMeta(meta, compactedMeta, windowRange)
 
-	fmt.Println("ID            : ", unifiedMeta.BlockID)
-	fmt.Println("Version       : ", unifiedMeta.Version)
-	fmt.Println("Total Objects : ", unifiedMeta.TotalObjects)
-	fmt.Println("Data Size     : ", humanize.Bytes(unifiedMeta.Size))
-	fmt.Println("Encoding      : ", unifiedMeta.Encoding)
-	fmt.Println("Level         : ", unifiedMeta.CompactionLevel)
-	fmt.Println("Window        : ", unifiedMeta.window)
-	fmt.Println("Start         : ", unifiedMeta.StartTime)
-	fmt.Println("End           : ", unifiedMeta.EndTime)
-	fmt.Println("Duration      : ", fmt.Sprint(unifiedMeta.EndTime.Sub(unifiedMeta.StartTime).Round(time.Second)))
-	fmt.Println("Age           : ", fmt.Sprint(time.Since(unifiedMeta.EndTime).Round(time.Second)))
+	//nolint:all
```

Comment:
what in the world did the linter not like about this?

Reply:
```
cmd/tempo-cli/cmd-list-block.go:74:46: unifiedMeta.Version undefined (type unifiedBlockMeta has no field or method Version) (typecheck)
	fmt.Println("Version       : ", unifiedMeta.Version)
```

Reply:
OH seems like they fixed it in their last release 8 days ago, will remove this